fvtest/compilertriltest/LogicalTest.cpp

```diff
@@ -59,6 +65,41 @@ TEST_P(Int32LogicalUnary, UsingConst) {
     ASSERT_EQ(exp, act);
 }
 
+TEST_P(Int64LogicalUnary, UsingConst) {
+    auto param = TRTest::to_struct(GetParam());
+
+    char inputTrees[120] = {0};
+    std::snprintf(inputTrees, 120, "(method return=Int64 (block (lreturn (%s (lconst %d) ) )))", param.opcode.c_str(), param.value);
```

Comment:
Can we format the trees in a tree-like manner as they would appear in trace files? See other tril tests for examples [1] [2].

[1] https://github.com/eclipse/omr/blob/2b09f123bdd94ccd44865ada5907f73dd692c227/fvtest/compilertriltest/IfxcmpgeReductionTest.cpp#L98-L114

[2] https://github.com/eclipse/omr/blob/2b09f123bdd94ccd44865ada5907f73dd692c227/fvtest/compilertriltest/MaxMinTest.cpp#L46-L56